packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -364,17 +364,26 @@ export const components: ThemeOptions["components"] = {
     },
   },
   MuiFormControl: {
+    defaultProps: {
+      margin: "normal",
+    },
```

Comment:
What's `margin: 'normal'`?

Reply:
Here, `margin` is a prop and not a CSS Property. There are `none`, `normal`, and `dense` variants in MUI.